R/inst/tutorials/01-playlist/README.md

```diff
@@ -11,8 +11,10 @@ in an R Markdown Notebook.**
 - Using the Metaflow client in an R Markdown Notebook.
 
 #### To play this episode:
-1. ```cd metaflow-tutorials/R/01-playlist/```
+1. ```cd tutorials/01-playlist/```
 2. ```Rscript playlist.R show```
 3. ```Rscript playlist.R run```
 4. ```Rscript playlist.R run --genre comedy```
-5. Open ```playlist.Rmd``` in RStudio.
\ No newline at end of file
+5. Open ```playlist.Rmd``` in RStudio.
```

Comment:
Do I need `RStudio` here? Do I have no other way here? Maybe we can restructure the instructions as
#### To play this episode
Execute the flow -
##### on terminal
1. blah blah
2. blah blah
##### on RStudio
1. blah blah
2. blah blah

Inspect the flow -
##### on terminal
1. blah blah
2. blah blah
##### on RStudio
1. blah blah
2. blah blah

Reply:
R Markdown is built into RStudio https://rmarkdown.rstudio.com/ so for the best experiences I believe R Studio should be the recommended way to open it.

Good idea on restructuring the instructions! Updating it now.